src/Lykke.AlgoStore.Core/Settings/ServiceSettings/ServiceSettings.cs

```diff
@@ -5,5 +5,6 @@ public class ServiceSettings
         public string SessionServiceUrl { get; set; }
         public string DeploymentApiServiceUrl { get; set; }
         public string AssetServiceUrl { get; set; }
+        public string ClientAccountServiceUrl { get; set; }
```

Comment:
I think it's better to use ClientAccountServiceClient as PersonalDataServiceClient, not to keep url separately, so in the settings file, we will append it as ClientAccountServiceClient.*. If client account url is changed, it will be changed in one place -> ClientAccountServiceClient section.  This is one of main Lykke conventions. We might need to change other url settings too.

Reply:
I agree with @dafinkagerdjikova 
Let us follow Lykke conventions so that won't have to change this in future